src/services/twitterRSSService.ts

```
@@ -0,0 +1,484 @@
+  }
+
+  private startScheduler(): void {
+    if (!this.isLoggedIn) {
+      logger.warn(
+        'Skipping RSS scheduler - Twitter authentication not available'
+      );
+      return;
+    }
+
+    const intervalMinutes = parseInt(
+      (this.getConfig('RSS_UPDATE_INTERVAL', '30') as string)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)